CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ From version 2.6.0, the sections in this file adhere to the [keep a changelog](h
 ### Fixed
 * [#2572](https://github.com/Shopify/shopify-cli/pull/2572): **Breaking** Update theme-check to 1.11.0 (dropped support for ruby 2.6)
 
+### Added
+* [#2600](https://github.com/Shopify/shopify-cli/pull/2600): Add support to the `SIGTERM` signal
```

Comment:
@karreiro sorry I didn't notice this before - this is in the wrong place, it'll be included in 2.25.0!